Assets/Scripts/TileMapDemoScripts/MapManager.cs

```diff
@@ -38,17 +39,16 @@ private void updateGameObjects()
     {
         Vector3Int tileLocation = getTileLocation();
 
-        if (Input.GetMouseButtonDown(0)){
+        if (Input.GetMouseButtonDown(0)) {
             TileBase clickedTile = tileMap.GetTile(tileLocation); // Get clicked tile
             player.updatePath(pathFinding.getPath(Vector3Int.FloorToInt(player.transform.position), tileLocation));
 
             BaseTile baseTile = (BaseTile)clickedTile;
             baseTile.debugPrintInformation();
         }
-
-        player.updateLocation(10.0f * Time.deltaTime);
```

Comment:
This breaks the intended functionality of the method. With the player's location not being updated, they can't be moved towards the pathfinded location. Did you intend to move this movement logic to its own method?